content/intro/getting-started/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Getting started with BluBracket"
 description: "Getting started with BluBracket"
```

Comment:
This really shouldn't be the same as the title (though it can be the same as the lead).